src/test/java/com/imprint/core/ImprintRecordBuilderTest.java

```diff
@@ -0,0 +1,234 @@
+package com.imprint.core;
+
+import com.imprint.error.ImprintException;
+import com.imprint.types.Value;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+class ImprintRecordBuilderTest {
```

Comment:
what's the difference between this test and ComprehensiveTest/IntegrationTest? I like that this one uses junit so we may as well just keep this one (and the tests below)